### ISSUE TITLE
Code Rewrite(?)

### ISSUE THREAD

**Issue by ghost** [arthurjdileo/virtual-classroom#14]
Because of how clunky the current code is in terms of OOP, I think we should rewrite the code in one of two ways:
# Version 1
class Whiteboard:
&nbsp;&nbsp;&nbsp;&nbsp;class Tools:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;class Brush
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;class Text
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;...
# Version 2
class Whiteboard:
&nbsp;&nbsp;&nbsp;&nbsp;...
class Tools:
&nbsp;&nbsp;&nbsp;&nbsp;class Brush
&nbsp;&nbsp;&nbsp;&nbsp;class Text
&nbsp;&nbsp;&nbsp;&nbsp;...

**Comment by ghost** [comment]
Additionally, we should create an event handler function or class on the server side to prevent clutter as we add more commands.

**Comment by arthurjdileo** [comment]
@mathewhan I agree with the above statements. I'd say we using Whiteboard as a wrapper class and then store all of its children inside of it.
